interp/src/stk_env.rs

```diff
@@ -872,8 +872,8 @@ mod priv_tests {
     #[test]
     fn value_shared_fork_point() {
         let mut smoosher = Smoosher::new();
-        smoosher.set("a", Value::try_from_init(2, 32).unwrap()); //for type inference
-                                                                 //the below fork adds a new scope to [smoosher]
+        smoosher.set("a", Value::from(2, 32).unwrap()); //for type inference
```

Comment:
why `from` and not `try_from_init`? I've just always been using `try_from_init`, is that not always the best choice?

Reply:
`try_from_init` was renamed to `from` It's the same method